### DISCUSSION TITLE
发现dolphinscheduler3.1.0具有以下bug，而且看3.1.2的release中也没有修复：

### DISCUSSION THREAD

**Discussion by qiux17** [apache/dolphinscheduler#13229] [General]
1.DS停止一个flink流处理的任务实例时，无法真正停止正在运行的flink任务；
2.定义工作流时，如果上下游任务之间存在传参的情形，则该工作流无法定时运行，会报空指针异常；
3.DS使用普通用户定义任务节点时，无法选择worker分组；
4.定义flink任务节点时，选择flink版本>1.12并保存，但是该配置最终却无法生效

**Comment by sunneebaby** [comment]
第一个3.1.3 spark任务实例也存在该问题，停掉任务实例，实际spark作业还在yarn上继续执行。
